ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/user_management_service.proto

```diff
@@ -98,6 +103,12 @@ message User {
     // Users for participant administrators MAY have an associated primary party.
     // Optional
     string primary_party = 2;
+
+    // Optional
+    bool is_deactivated = 3;
+
+    // Optional
+    repeated Property properties = 4;
```

Comment:
I'd suggest that we define a reduced `ObjectMetadata` type inspired by the `ObjectMetadata` from https://github.com/kubernetes/apimachinery/blob/master/pkg/apis/meta/v1/generated.proto#L640, but including only the `annotations` field with the exact same type (map<string,string>) and fieldnumber (12) as the one used by Kubernetes.

The references to that are then done the same way as for k8s, i.e. using a `ObjectMetadata metadata` field on users and parties.

Reply:
This way we open up the avenue for taking over more of the k8s concepts in case a need arises.

Reply:
Ok